Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:lts
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
```

Comment:
Why is this done in separate steps rather than copying everything then running npm install?

Reply:
The reason as to why I am not copying all of the contents before is so that Docker can cache those steps. If you make changes to the application and rebuild it, it will not run through those steps again.